### DISCUSSION TITLE
Url for pages

### DISCUSSION THREAD

**Discussion by mraf** [givanz/Vvveb#129] [General]
Hi, 

is there a chance to skip /pages in urls or is the way to manage redirections or make friendly urls? 

I'm asking as current website have structure like this and it's well ranked with google:

for example

/about
/gallery
/faq

and more..

so basically all pages :)

**Comment by givanz** [comment]
Hi

You need to edit `config/routes.php`

Pages have `/page/{slug}` and blog posts have no prefix `/{slug}`
 
You need to swap page configuration with post.
 
Now it's like this

```php 
'/{slug}'     => ['module' => 'content/post/index', 'edit'=>'index.php?module=content/post&slug={slug}&type=post'],
//page
//'/{slug}'        => ['module' => 'content/page/index', 'edit'=>'index.php?module=content/post&slug={slug}'],
'/page/{slug}' => ['module' => 'content/page/index', 'edit'=>'index.php?module=content/post&slug={slug}&type=page'],
```

You need to change it to

```php 
'/page/{slug}' => ['module' => 'content/post/index', 'edit'=>'index.php?module=content/post&slug={slug}&type=post'],
//page
//'/{slug}'    => ['module' => 'content/page/index', 'edit'=>'index.php?module=content/post&slug={slug}'],
'/{slug}'     => ['module' => 'content/page/index', 'edit'=>'index.php?module=content/post&slug={slug}&type=page'],
 ```
 
You might need to clear cache after the change.
